next.config.js

```diff
@@ -9,8 +9,10 @@ import { withSentryConfig } from "@sentry/nextjs";
 const nextConfig = {
   productionBrowserSourceMaps: true,
   sentry: {
-    disableServerWebpackPlugin: process.env.UPLOAD_SENTRY_SOURCEMAPS ?? "false",
-    disableClientWebpackPlugin: process.env.UPLOAD_SENTRY_SOURCEMAPS ?? "false",
+    disableServerWebpackPlugin:
+      process.env.UPLOAD_SENTRY_SOURCEMAPS === "true" ? false : true,
```

Comment:
Guessing we wouldn't want something like `process.env.UPLOAD_SENTRY_SOURCEMAPS !== "true"` and avoiding the ternery.

<s>And probably a temp variable is overkill too if you wanted
```js
const uploadSentrySourcemaps = process.env.UPLOAD_SENTRY_SOURCEMAPS === "true";

disableServerWebpackPlugin: !uploadSentrySourcemaps,
disableClientWebpackPlugin: !uploadSentrySourcemaps,
```
</s>

Reply:
PS: let me know if you want me to quickly R+ this if you want to get it onto Stage/Prod right meow and just want a blind 👍 to appease the CI Gods.

Reply:
Good idea, thanks! I figured out a way to test this before landing (I could test some it locally, but I want to make sure it's working when run from CircleCI), the feedback cycle is so long if we have to push to stage to test: https://github.com/mozilla/blurts-server/pull/4179

Reply:
(so no rush to stage but I appreciate the offer!)

Reply:
No problem. Let the record show I have poor ethics and am willing to blind R+ at 9PM for the sake of :yolo: based development (and the LOLs). 🤷